src/core/ProjectRenderer.cpp

```diff
@@ -103,8 +102,6 @@ ProjectRenderer::ProjectRenderer( const Mixer::qualitySettings & qualitySettings
 
 ProjectRenderer::~ProjectRenderer()
 {
-	Engine::mixer()->restoreAudioDevice();  // also deletes audio-dev
-	Engine::mixer()->changeQuality( m_oldQualitySettings );
```

Comment:
Make sure #3680 still works. If [these test projects](https://github.com/LMMS/lmms/pull/3680#issuecomment-312736420) renders, as instructed, it's a 'go' from me.

Reply:
@zonkmachine I'm pretty sure #3680 will still work because `RenderManager` destructor is called after `ProjectRenderer` destructor. However, testing is always good. I'll double-check it!

Reply:
@zonkmachine Tested. It still works!

Reply:
OK. I think you should merge.